### DISCUSSION TITLE
How to open link on new tab?

### DISCUSSION THREAD

**Discussion by igorroc** [remarkjs/remark#1135] [Q&A] [answered]
How do I configure so every [link](https://google.com) opens with `target="_blank"` ?

**Comment by ChristianMurphy** [comment, marked as answer]
Heya @igorroc 👋 
Generally the convention is external links are `_blank`, and internal links open in the same tab.
This can be done through https://github.com/remarkjs/remark-external-links

If you want every link to open in a new tab, you could create your own plugin (more on that here: https://unifiedjs.com/learn/) and take inspiration from how `remark-external-links` uses `hProperties` https://github.com/syntax-tree/mdast-util-to-hast#fields-on-nodes

**Reply by igorroc** [reply]
But I've tested and even external links are opening in the same tab

**Reply by ChristianMurphy** [reply]
Can you share an example of what you tried?
Trying out the plugin in a sandbox like: https://codesandbox.io/s/remark-external-link-y7dmz7 and https://codesandbox.io/s/remark-external-links-joz83m both work as expected.

**Comment by Khanon** [comment]
Hello @ChristianMurphy , how can I make it works in a *react-markdown* element?

**Reply by ChristianMurphy** [reply]
Same as the selected answer 🙂 
https://github.com/orgs/remarkjs/discussions/1135#discussioncomment-5339294
